pages/glossary/distinguishable-styles.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Distinguishable Styles
+key: distinguishable-styles
+unambiguous: true
+objective: true
+---
+
+Two elements have _distinguishable styles_ if at least one of the following is true:
+
+- they have [distinguishable colors](#distinguishable-colors); or
+- they have [distinguishable borders](#distinguishable-borders); or
+- they have [distinguishable box-shadows](#distinguishable-box-shadows); or
+- they have different [computed values][] for the `background-image` property; or
+- they have different [computed values][] for at least one [text style property](#text-style-properties).
+
+This definition assumes that different [computed values][] for the `background-image` property specify different images. If one [computed value][computed values] is a data URI and the other is a URL of the same image then this definition will produce incorrect results.
+
+[computed values]: https://www.w3.org/TR/css-cascade/#computed-value 'Definition of computed values'
```

Comment:
This definition doesn't allow for `outline` as a distinguishing style, which doesn't seem entirely appropriate. Consider this:

```html
<style>
  a {
    text-decoration: none;
  }

  a:hover,
  a:focus {
    outline: 1px auto;
  }
</style>

<p>Hello <a href="#">link</a></p>
```

As far as I can tell, the above would fail the rule.

Reply:
`outline` is part of "distinguishable borders": https://github.com/act-rules/act-rules.github.io/pull/1010/files#diff-6483e6f55c8ac9e11ed6fbaddc5cf2867e42ca7d77da69d9764512064be4152fR11

Reply:
(well, sort of because then the def looks at the associated longhand which `outline` doesn't have…)

Reply:
Oh, I completely missed that! 🤦